src/components/Toast/Toast.module.css

```diff
@@ -1,170 +1,46 @@
-  color: var(--eds-theme-color-utility-error-foreground);
-}
+  padding-left: var(--eds-size-4);
+  padding-right: var(--eds-size-1);
+  
+  display: grid;
+  gap: var(--eds-size-1);
+  grid-template-columns: 1fr min-content;
+  align-items: center;
+  
+  border: var(--eds-border-width-sm);
+  border-left-width: var(--eds-border-width-lg); /* 3 */
+  border-radius: var(--eds-size-half);
 
-/**
- * Brand toast
- */
-.toast--brand {
-  background: var(--eds-color-brand-blue-100); /* TODO */
-  border-color: var(--eds-theme-color-primary-border);
-  border-left-color: var(--eds-theme-color-primary-foreground); /* 1 */
-  color: var(--eds-theme-color-primary-foreground);
+  border-color: var(--border-light-color);
+  border-left-color: var(--border-dark-color);
+  
+  box-shadow: var(--eds-box-shadow-sm);
+  --svg-icon-size-default: 1.25rem;
 }
 
 /**
- * Toast informational icon
- * 1) Selector doubled up to bump specificity to not interact with .icon.
- *    See code guidelines for more info.
+ * 1) The toast content, usually an icon and text message. Does not include the optional close button
+ * 2) Grid aligns the centers and places the icon on the left and message on the right.
  */
-.toast__icon.toast__icon {
-  margin-right: var(--eds-size-2);
-  fill: var(--eds-theme-color-neutral-medium-foreground);
-  width: var(--eds-size-3);
-  height: var(--eds-size-3);
-
-  /**
-   * Icon within success toast
-   */
-  .toast--success & {
-    fill: var(--eds-theme-color-utility-success-foreground);
-  }
-
-  /**
-   * Icon within warning toast
-   */
-  .toast--warning & {
-    fill: var(--eds-theme-color-utility-warning-foreground);
-  }
-
-  /**
-   * Icon within error toast
-   */
-  .toast--error & {
-    fill: var(--eds-theme-color-utility-error-foreground);
-  }
-
-  /**
-   * * Icon within info toast
-   */
-  .toast--info & {
-    fill: var(--eds-theme-color-utility-info-foreground);
-  }
-
-  /**
-   * * Icon within info toast
-   */
-  .toast--brand & {
-    fill: var(--eds-theme-color-primary-foreground);
-  }
-}
-
-/**
- * Toast body
- * 1) Allows the body to take up the remaining space that the icon doesn't occupy
- */
-.toast__body {
-  flex: 1;
-}
-
-/**
- * Toast title
- */
-.toast__title {
-  @mixin eds-theme-typography-heading-3;
-  margin-bottom: var(--eds-size-1);
-  flex: 1 0 auto;
-
-  @media all and (min-width: $eds-bp-md) {
-    margin-right: var(--eds-size-6);
-  }
-}
-
-/**
- * Toast description
- */
-.toast__description {
-  @mixin eds-theme-typography-body-text;
-}
-
-/**
- * Toast close button
- * 1) Button used to dismiss dismissable toasts
- * 2) Selector doubled up to bump specificity to not interact with .icon.
- *    See code guidelines for more info.
- * 3) Match close button icon color to the toast text color
- */
-.toast__close-btn.toast__close-btn {
-  position: relative;
-  top: var(--eds-size-half);
-  margin-left: var(--eds-size-3);
-  color: inherit; /* 3 */
-}
+.toast--content { /* 1 */
```

Comment:
suuuuper nit but if a comment refers to the whole block, I think it could go at the top of the comment block and not have a number. Like "Toast information icon" in the old file